packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -556,7 +571,6 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
     const assembly = await assemblyFromSource(cx, false);
     const ioHost = withAction(this.ioHost, 'watch');
     const rootDir = options.watchDir ?? process.cwd();
-    await ioHost.notify(debug(`root directory used for 'watch' is: ${rootDir}`));
```

Comment:
why remove these comments from watch?

Reply:
They are moved further down to be emitted as a single message.